src/js/category.js

```diff
@@ -0,0 +1,256 @@
+const listProduct = document.querySelector('.category__product-list');
+const gridProduct = document.querySelector('.category__product-grid');
+const menuCategory = document.querySelector('.filter__nav');
+let categoryApplied = '';
```

Comment:
```suggestion
let filterApplied = {};
```
object em nhé, để còn apply nhiều thứ như page, limit, sort,... nữa